### ISSUE TITLE
System.Runtime.InteropServices.COMException: Es wird bereits ein Ziehvorgang ausgeführt.

### ISSUE THREAD

**Issue by pepperb0i** [RandomEngy/VidCoder#1295]
### What were you doing when VidCoder crashed?

Dragging and dropping videos onto the vidcoder surface when the crash warning popped up all of a sudden.

### Crash details

```text
VidCoder 10.14 VelopackInstaller
System.Runtime.InteropServices.COMException (0x80004005): Es wird bereits ein Ziehvorgang ausgeführt.

   at MS.Win32.UnsafeNativeMethods.DoDragDrop(IDataObject dataObject, IOleDropSource dropSource, Int32 allowedEffects, Int32[] finalEffect)
   at System.Windows.OleServicesContext.OleDoDragDrop(IDataObject dataObject, IOleDropSource dropSource, Int32 allowedEffects, Int32[] finalEffect)
   at System.Windows.DragDrop.OleDoDragDrop(DependencyObject dragSource, DataObject dataObject, DragDropEffects allowedEffects)
   at System.Windows.DragDrop.DoDragDrop(DependencyObject dragSource, Object data, DragDropEffects allowedEffects)
   at VidCoder.Controls.PresetTreeViewContainer.OnPresetTreeItemMouseMove(Object sender, MouseEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.MouseDevice.Synchronize()
   at System.Windows.Input.InputManager.HitTestInvalidatedAsyncCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by RandomEngy** [comment]
Thanks for the report. This crash should be fixed in the next Beta version.

**Comment by RandomEngy** [comment]
Fixed in 11.7 Beta.
